File: patterns/c-patterns/pattern114.c
// Pattern 114: ...

//     *****
//    *****
//   *****
//  *****
// *****

#include <stdio.h>

int main()
{
    int z = 1;
    int n = 5;
    int i, j, k;

    for (i = 0; i < n; i++)
    {
        for (j = n - 1; j > i; j--)
        {
            printf(" ");
        }
        for (k = 0; k < n; k++)
        {
            printf("*");
        }
        z++;
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern114.js
// Pattern 114: ...

//     *****
//    *****
//   *****
//  *****
// *****

let z = 1;
let n = 5;

for (let i = 0; i < n; i++) {
    let spaces = "";
    let stars = "";

    for (let j = n - 1; j > i; j--) {
        spaces += " ";
    }

    for (let k = 0; k < n; k++) {
        stars += "*";
    }

    console.log(spaces + stars);

    z++;
}
